jetstream/core/orchestrator.py

```diff
@@ -223,6 +223,7 @@ def __init__(
       interleaved_mode: bool = False,
       jax_padding: bool = True,
       metrics_collector: JetstreamMetricsCollector | None = None,
+      ray_multiple_host: bool = False,
```

Comment:
nit: a better name for this could be with_ray_backend?

Reply:
Sounds good and thanks the suggestion! With_ray_backend is better name, then I feel is_ray_backend (we usually use 'is' or 'enable' for bool type parameter prefix). 

Changed to is_ray_backend, how about this name?

Reply:
sg! Thanks